ui/app/store/actions.js

```diff
@@ -2454,17 +2458,25 @@ export function rejectPermissionsRequest(requestId) {
  * Clears the given permissions for the given origin.
  */
 export function removePermissionsFor(domains) {
-  return () => {
-    background.removePermissionsFor(domains);
+  return (dispatch) => {
+    background.removePermissionsFor(domains, (err) => {
+      if (err) {
+        dispatch(displayWarning(err.message));
+      }
+    });
```

Comment:
adding error handling to a couple places here. could be the source of some uncaught errors?